### ISSUE TITLE
12Gb GPU

### ISSUE THREAD

**Issue by nachoXXL** [luca-medeiros/lightning-sam#25]
Hi all,
A question I've a rtx 2080 ti with 12 Gb where only one image (embedding) fits.
So, as a start I could use a batch size of one but this does not train well.

Would it be an option to run several images sequentially before doing the backward step?

something like:
for epoch
   loss = tensor
  for image in ["N images"]
      loss+ = predict img
      loss += predict img
      loss += predict img
      loss += predict img
  backward(loss)

Regards,
Nacho
